tests/test_symbol.py

```diff
@@ -135,6 +136,10 @@ def test_by_name(self):
                 self.assert_symbol_equal(prog.symbol("second"), second)
                 self.assertRaises(LookupError, prog.symbol, "third")
 
+                self.assert_symbol_equal(prog.symbols("first")[0], first)
+                self.assert_symbol_equal(prog.symbols("second")[0], second)
```

Comment:
This should probably check that the list is length 1, too. Maybe add a `self.assert_symbols_equal()` method that compares a `List[drgn.Symbol]` to a `List[Symbol]` rather than hard-coding it?